### DISCUSSION TITLE
Comparison OpenDTU vs ahoy

### DISCUSSION THREAD

**Discussion by timguy** [tbnobody/OpenDTU#362] [General]
Anybody know how these both projects OpenDTU and [ahoy](https://github.com/lumapu/ahoy/tree/main/tools/esp8266) compare to each other?
Ahoy also mention OpenDTU as "sister project" but code is not shared in a sense of libs or anything else?
I only saw that ahoy will also run on ESP8266 and has a different looking web interface.
So anybody did a comparison for hard facts and features?

**Comment by stefan123t** [comment]
Hi @timguy we have been sharing some code in the past and @tbnobody is a frequent visitor to our Discord chat. Currently the AhoyDTU supports both hardware platforms and historicly we have some relicts from the early Proof of Concept code from the microcontroller forum hosted in our AhoyDTU repository.
Regarding features I currently know if the AlarmData/AlarmUpdate commamd which is supported in OpenDTU which isn't implemented in AhoyDTU yet.
There is a fork of an early AhoyDTU which supports MI-Model inverters which should be integrated in AhoyDTU and the aim is to build a library of common functions / methods for both Gen 2 MI- and Gen 3 HM- inverters. We do not know yet what will happen / be supported with HMS-/HMT- inverters hitting the European markets since this year only.
